package.json

```diff
@@ -35,6 +35,8 @@
     "@octokit/rest": "^16.43.2",
     "@types/enzyme": "3.9.0",
     "@types/jest": "27.0.2",
+    "@types/react": "^17.0.0",
```

Comment:
I've added these types to the root package so that it is shared explicitly for all workspaces instead of implicitly trough some packages that happen to include it.